src/WarmUp/UrlsPattern/CategoryCmsPage.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\WarmUp\UrlsPattern;
+
+use Magento\MagentoCloud\WarmUp\UrlsPattern;
+
+/**
+ * Processes cms-page and category pattern types.
+ */
+class CategoryCmsPage implements PatternInterface
+{
+    /**
+     * @var ConfigShowUrlCommand
+     */
+    private $configShowUrlCommand;
+
+    /**
+     * @var CommandArgumentBuilder
+     */
+    private $commandArgumentBuilder;
+
+    /**
+     * @param ConfigShowUrlCommand $configShowUrlCommand
+     * @param CommandArgumentBuilder $commandArgumentBuilder
+     */
+    public function __construct(
+        ConfigShowUrlCommand $configShowUrlCommand,
+        CommandArgumentBuilder $commandArgumentBuilder
+    ) {
+        $this->configShowUrlCommand = $configShowUrlCommand;
+        $this->commandArgumentBuilder = $commandArgumentBuilder;
+    }
+
+    /**
+     * @param string $pattern can accept the next value types:
+     * - "*" - all pages
+     * - "regular expression" - pages will be filtered by this pattern
```

Comment:
`"regular expression"` - looks like a string value. Maybe `{regular expression}` is better/clear?

Reply:
Agree, changed